packages/target-tools/src/telemetryProvider.spec.js

```diff
@@ -1,7 +1,6 @@
 import { TelemetryProvider } from "./telemetryProvider";
```

Comment:
We'll probably need to add another test case or two in order to test the logic inside `getMode`.  The test(s) will be similar to your existing addEntry test because we don't want to export getMode.